### PR TITLE
Added Amazon Clone Frontend

### PR DESCRIPTION
**Pull request by shellygarg10**
Closes:  #714

- **Title** :  Amazon Clone Frontend
- **Name:** :  Shelly Garg
- **Idenitfy yourself:** :  GSSOC 24

### Describe the add-ons or changes you've made 📃
Added a project made using basics of HTML and CSS.

### Screenshots 📷

image

### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Checklist: ☑️

- [x] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [x] This PR does not contain plagiarized content.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] My changes generate no new warnings.

### CODE REVIEW

**Comment by shellygarg10** [comment]
@Avdhesh-Varshney can you please explain the issues to me

**Comment by shellygarg10** [comment]
@Avdhesh-Varshney i made corrections and followed all the guidelines correctly now , please review my pr once again

File: Front-end-Projects/Basic/Amazon-Clone/Amazon_clone.png
Comment (Avdhesh-Varshney) [inline Front-end-Projects/Basic/Amazon-Clone/Amazon_clone.png:1]: convert it into `webp` format and rename it to `screenshot`.

File: Front-end-Projects/README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@
 |   20  | [Login Signup Form](./Basic//Login-Signup-Form) | ![Basic](https://img.shields.io/badge/Basic-00FF00?style=for-the-badge) |
 |   21  | [Feedback Form](./Basic//Feedback-Form) | ![Basic](https://img.shields.io/badge/Basic-00FF00?style=for-the-badge) |
 |   22  | [Survey Form](./Basic//Survey-Form) | ![Basic](https://img.shields.io/badge/Basic-00FF00?style=for-the-badge) |
-
+|   23  | [Amazon Clone](./Basic//Amazon_clone) | ![Basic](https://img.shields.io/badge/Basic-00FF00?style=for-the-badge) |
```
Comment (Avdhesh-Varshney) [inline Front-end-Projects/README.md:50]: Correct here also.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]

File: Front-end-Projects/Basic/Amazon-Clone/Amazon_clone.png
Comment (Avdhesh-Varshney) [reply Front-end-Projects/Basic/Amazon-Clone/Amazon_clone.png:1]: Don't forget to correct your image in your project readme file.

**Pull request closed without merging**